starwars/src/components/Card.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import "./StarWars.css";
+import luke from "../images/luke.jpg";
+import c3po from "../images/c3po.jpg";
+import r2d2 from "../images/r2d2.jpg";
+import vader from "../images/vader.jpg";
+import leia from "../images/leia.jpg";
+import lars from "../images/lars.jpg";
+import beru from "../images/beru.jpg";
+import r5d4 from "../images/r5d4.jpg";
+import biggs from "../images/biggs.jpg";
+import obi from "../images/obi.jpg";
+
+const Card = props => {
+  // let charPic = props.picture.map(pic => pic.name);
+  let charPic = [luke, c3po, r2d2, vader, leia, lars, beru, r5d4, biggs, obi];
+  console.log(charPic);
+
+  return (
+    <div className="container">
+      {props.starwars.map((char, i) => {
+        return (
+          <div className="character">
```

Comment:
Seems like code could be made drier here with a cleaver use of `Object.keys` to provide a way to map through the array of attributes to generate the `p` and `strong` jsx tags.